src/internal/connector/sharepoint/site_page.go

```diff
@@ -0,0 +1,93 @@
+package sharepoint
```

Comment:
Since we'll be moving this direction anyway, how about migrating this file into `/sharepoint/api/pages.go`?  No other changes needed.  Wrapping an API client can happen later.  But it saves us a step in the future from migrating the file.

Reply:
Several changes were needed to drop into new file. Resolved with the final commint [a088da7](https://github.com/alcionai/corso/pull/2178/commits/a088da7517172ed203f02645d1cff04ce455c7cf)